tools/wurf_options.py

```diff
@@ -63,6 +66,20 @@ def resolve(ctx):
 
 class WurfOptions(Options.OptionsContext):
 
+    def execute(self):
+
+        # Execute the context, this will go out and invoke the option(...)
+        # function defined in the wscript. We have some extra tools that
+        # also define the option(...) function to avoid having to call load
+        # in all wscripts to initialize those, we can do it here.
+
+        self.load('wurf_common_tools')
```

Comment:
I think the answer is no to my last question. Your changes only hide the load operation for options and resolve. It just feels weird that you have to load the tool manually in some steps, but in some other steps it is loaded for you automagically. I would rather use the explicit load operation at all places!
The explicit solution would also allow us to *not* load all the common tools where we don't need them.